### ISSUE TITLE
Failed to create the native project - Delete the ios and or android folders... I have none to begin with?

### ISSUE THREAD

**Issue by technologyconsulting** [realm/realm-js#6482]
### How frequently does the bug occur?

Always

### Description

Trying to follow the guide: https://www.mongodb.com/docs/realm/sdk/react-native/bootstrap-with-expo/

However, when I get to the 'expo run:android' part I get 

Failed to create the native project
You may want to delete the `./ios` and/or `./android` directories before trying again.

But, I have no ios or android folder to begin with?

### Stacktrace & log output

```shell
Mon Feb 12~ExpoRealm (master)>npx expo run:android

> exporealm@1.0.0 android
> expo run:android

env: load .env.local
env: export REACT_NATIVE_PACKAGER_HOSTNAME
(node:42476) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
env: load .env.local
env: export REACT_NATIVE_PACKAGER_HOSTNAME
spawn cmd.exe ENOENT
✖ Failed to create the native project.
You may want to delete the `./ios` and/or `./android` directories before trying again.
```

### Can you reproduce the bug?

Always

### Version

Expo 49.0.8

### What services are you using?

Local Database only

### Are you using encryption?

No

### Platform OS and version(s)

Windows 11

### Build environment

Which debugger for React Native: ..

**Comment by kneth** [comment]
On` MacOS, the template app works fine, and we are currently trying to reproduce it on Windows.

**Comment by technologyconsulting** [comment]
Hey Kenneth,

I knew I forgot to update something :-) I figured out the issue (workaround).

I needed to run the following: npm install expo@latest && npx expo install --fix

Firing up a fresh install of expo and running a standard expo template worked ok. When trying the realm template from the documentation didn't work and showed an outdated expo-cli version? Running the command above has solved this but there was no mention in the docs.
